src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/respondentsreport/RespondentsReport.java

```diff
@@ -81,19 +81,32 @@ private boolean hasMultipleRespondents(RespondentsReportCaseData caseData) {
     }
 
     private boolean isRepresentativeRepresentingMoreThanOneRespondent(String rep, RespondentsReportCaseData caseData) {
+        if (CollectionUtils.isEmpty(caseData.getRepCollection())) {
+            return false;
+        }
+
+        int count = countRepresentedRespondents(rep, caseData);
+        return count > 1;
+    }
+
+    private int countRepresentedRespondents(String rep, RespondentsReportCaseData caseData) {
         int count = 0;
-        if (CollectionUtils.isNotEmpty(caseData.getRepCollection())) {
-            for (RepresentedTypeRItem repItem : caseData.getRepCollection()) {
-                if (repItem.getValue().getNameOfRepresentative().equals(rep)) {
-                    for (RespondentSumTypeItem respItem : caseData.getRespondentCollection()) {
-                        if (respItem.getValue().getRespondentName().equals(repItem.getValue().getRespRepName())) {
-                            count++;
-                        }
-                    }
-                }
+        for (RepresentedTypeRItem repItem : caseData.getRepCollection()) {
+            if (repItem.getValue().getNameOfRepresentative().equals(rep)) {
+                count += countMatchingRespondents(repItem.getValue().getRespRepName(), caseData);
             }
         }
-        return count > 1;
+        return count;
+    }
```

Comment:
Github won't let me make a suggestion here, but using streams might make this tidier

```
private int countRepresentedRespondents(String rep, RespondentsReportCaseData caseData) {
    return caseData.getRepCollection().stream()
            .filter(repItem -> repItem.getValue().getNameOfRepresentative().equals(rep))
            .mapToInt(repItem -> countMatchingRespondents(repItem.getValue().getRespRepName(), caseData))
            .sum();
}
```